modules/browser/BrowserCapabilities.js

```diff
@@ -226,6 +226,17 @@ export default class BrowserCapabilities extends BrowserDetection {
         return !this.isFirefox();
     }
 
+    /**
+     * Returns true if the browser supports the new Scalability Mode API for VP9/AV1 simulcast. H.264 simulcast will
+     * also be supported by the jvb for this version because the bridge is able to read the Dependency Descriptor RTP
+     * header extension to extract layers information for H.264 as well.
+     *
+     * @returns {boolean}
+     */
+    supportsScalabilityModeAPI() {
```

Comment:
Can't this be detected at runtime somehow?

Reply:
I am not sure if we want to enable it based on feature detection. I think its better to go with a version that we have tested and we know works as expected since its available in 113 onwards but there were bugs . WDYT?

Reply:
Gotcha, makes sense!